packages/patternlab/styleguide/source/assets/js/modules/mapLeaflet.js

```diff
@@ -69,7 +69,7 @@ export default (function (window,document) {
 
       // custom marker icon 
       var markerIcon = L.icon({
-        iconUrl: `${ma.iconPath}marker-blue.svg`,
+        iconUrl: `${ma.iconPath}/marker-blue.svg`,

```

Comment:
@vordude this changed broke the path in patternlab
https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-20949-google-map-swap-poc/?p=pages-location-general-content
![Screen Shot 2021-04-28 at 2 16 10 PM](https://user-images.githubusercontent.com/5789411/116453004-58656700-a82c-11eb-8804-4499709d9a88.png)